airflow/api_fastapi/core_api/datamodels/backfills.py

```diff
@@ -56,3 +57,15 @@ class BackfillCollectionResponse(BaseModel):
 
     backfills: list[BackfillResponse]
     total_entries: int
+
+
+class BackfillRunInfo(BaseModel):
+    """Data model for run information during a backfill operation."""
+
+    logical_date: datetime
```

Comment:
We will only have logical date, no data interval?

Reply:
Since the logical date is a datetime, so for example for DAGs with schedules other than daily, the response will still include the dates on which the backfill will be triggered. I imagine that the user likely has an understanding of the dag's schedule when making a backfill request. Does including the data interval provide additional value in this context?

<img width="472" alt="image" src="https://github.com/user-attachments/assets/3d781536-48b7-499e-b01b-80733f6a5ff3" />